src/pages/components/CertificateInfo.qml

```diff
@@ -103,146 +105,69 @@ SilicaFlickable {
-        }
-
-        CertificateLabel {
-            //: Label for the issuer field of a TLS certificate
-            //% "Issuer"
-            key: qsTrId("sailfish_browser-la-cert_issuer")
-            value: security ? security.issuerDisplayName : ""
-            keyWidth: root.keyWidth
+        Button {
+            anchors.horizontalCenter: parent.horizontalCenter
+            //% "Certificate details"
+            text: qsTrId("sailfish_browser-sh-certificate_details")
+            enabled: security && !security.certIsNull
+            visible: enabled
+            onClicked: showCertDetail()
         }
 
-        CertificateLabel {
-            //: Label for the TLS certificate validity period (dates)
-            //% "Valid"
-            key: qsTrId("sailfish_browser-la-cert_valid_dates")
-            value: insecure ? Format.formatDate(security.effectiveDate, Formatter.DateMedium)
-                            + " — "
-                            + Format.formatDate(security.expiryDate, Formatter.DateMedium)
-                          : ""
-            insecure: security && !security.certIsNull && security.notValidAtThisTime
-            keyWidth: root.keyWidth
-        }
+        Column {
+            width: parent.width
+            Label {
+                width: parent.width - 2 * Theme.horizontalPageMargin
+                x: Theme.horizontalPageMargin
+                horizontalAlignment: Text.AlignHCenter
+                wrapMode: Text.Wrap
+                color:  Theme.highlightColor
+                visible: permissionIndicationModel.count > 0
+                //% "Current permissions"
+                text: qsTrId("sailfish_browser-sh-current-permissions")
+            }
 
-        CertificateLabel {
-            //: Label prefixing the status of whether tracking tech is being used on a page
-            //% "Tracking"
-            key: qsTrId("sailfish_browser-la-cert_tracking")
-            value: {
-                if (security) {
-                    if (security.loadedTrackingContent) {
-                        //% "The page includes items used to track your behaviour"
-                        return qsTrId("sailfish_browser-la-cert_tracking_content_loaded")
-                    } else if (security.blockedTrackingContent) {
-                        //% "Items used to track your behaviour have been blocked"
-                        return qsTrId("sailfish_browser-la-cert_tracking_content_blocked")
+            Grid {
+                rows: permissionIndicationModel.count > 5 ? 1 : 2
+                columns: permissionIndicationModel.count
+                anchors.horizontalCenter: parent.horizontalCenter
+                spacing: Theme.paddingMedium
+                Repeater {
+                    model: permissionIndicationModel
+                    delegate: Icon {
+                        source: model.iconSource
+                        color: Theme.highlightColor
+                        Icon {
```

Comment:
There is a reason why I could not do this. When opening site permissions, the user sees the settings for possible permissions, even if they are not in the list, he can manage them. For example, it can deny permission for cookies, but by default it will not be displayed in the indication of current permissions.

Reply:
Yes, I can see what you mean. You could have the icons clickable through to the site permissions dialogue, and only show the "Site permissions" button if there are none to click?